CONTRIBUTING.md

```diff
@@ -56,6 +64,7 @@ TiDB 中文文档的日常更新特别活跃，相应地，[TiDB 英文文档](h
 - [TiDB 中文用户文档模板](/resources/tidb-docs-template-zh-v1.0.pdf)
 - [必须遵循的 Markdown 规范](#必须遵循的-markdown-规范)
 - [代码注释规范](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
+- [Github 常用命令行](https://docs.google.com/document/d/1slyV7ODvMs3QjfLkFpNCCO0e0BBlDF4MZdPc3yQO_RU)
```

Comment:
I think contributors may want to know more about the commands we mentioned in the translation tasks claim process and there are many useful cases in this google doc, so I added this line. But I am not sure whether we can share this google doc to the public because then everyone can edit it. Or maybe we can lock this google doc?

Reply:
We do not share internal google docs with the public. Maybe you can add a table somewhere that introduces the commands you mentioned earlier. @CharLotteiu